packages/SwingSet/bin/vat

```diff
@@ -28,8 +28,9 @@ async function main() {
   const vatArgv = argv[0] === '--' ? argv.slice(1) : argv;
 
   const config = await loadBasedir(basedir);
-  const ldSrcPath = require.resolve('../src/devices/loopbox-src');
-  config.devices = [['loopbox', ldSrcPath, {}]];
+  const { buildLoopbox } = require('../src/devices/loopbox');
```

Comment:
Not immediately germane to this change, but is there any deep reason why this is CJS code?

Reply:
meh, not really. It's super old, probably the first swingset file I added. And to make it use `import` I would have had to split it into two files, or use that not-so-portable `#!/usr/bin/env -S` trick that I didn't even know about back then.